Vagrantfile

```diff
@@ -58,20 +58,27 @@ Vagrant.configure("2") do |config|
         config.vagrant.plugins = required_plugins
     end
 
-	config.vm.synced_folder ".", "/vagrant", disabled: false, type: 'rsync'
+    config.vm.synced_folder '.', CONFIGURATION['base_folder'], disabled: false, type: 'rsync'
+    config.vm.provision 'file', source: 'configuration.yml', destination: CONFIGURATION['base_folder'] + '/web_s2i/configuration.yml'
 
   config.vm.provision "docker" do |d|
-    d.build_image "/vagrant/docker",
-      args: "-t devopsloft/devopsloft"
+    d.post_install_provision "shell", inline:"docker network create devopsloft_network"
+    d.build_image CONFIGURATION['base_folder'] + '/db_s2i',
+      args: '-t ' + CONFIGURATION['dockerhub_organization'] + '/mysql --build-arg MYSQL_ROOT_PASSWORD=' + CONFIGURATION['mysql_root_password'] + ' --build-arg MYSQL_DATABASE=' + CONFIGURATION['mysql_db']
+    d.run "db",
+      image: CONFIGURATION['dockerhub_organization'] + '/mysql',
```

Comment:
be consistent with the web image name. Or, use a configuration attribute here also or use in the web a hard coded name

Reply:
I understand that you mean to adjust db image variables to be like the web image format
so it will be:
CONFIGURATION['dockerhub_organization'] + '/' + CONFIGURATION['dockerhub_**db**']
OK?